unittest/mjcf.cpp

```diff
@@ -923,7 +924,8 @@ BOOST_AUTO_TEST_CASE(adding_keyframes)
                     <key name="test"
                     qpos="0 0 0.596
                         0.988015 0 0.154359 0
-                        0.988015 0 0.154359 0"/>
+                        0.988015 0 0.154359 0
+                        "/>
```

Comment:
Should I revert this change? It's just meant as an extra test, otherwise a similar case is added in the `test_get_unknown_size_vector_from_stream` test